grafana-plugin/src/containers/UserSettings/parts/tabs/GoogleCalendar/GoogleCalendar.tsx

```diff
@@ -43,111 +41,141 @@ const GoogleCalendar: React.FC<{ id: ApiSchemas['User']['pk'] }> = observer(({ i
   };
 
   const handleSchedulesChange = (value) => {
-    setGoogleCalendarSettings((v) => ({ ...v, oncall_schedules_to_consider_for_shift_swaps: value }));
-
     userStore.updateCurrentUser({
```

Comment:
I'm not up-to-date with the GCalendar features, aren't these settings available to multiple user views? It seems this call only updates the current user, which seems to be always the one logged in

Reply:
good catch! anyway it's more correct
```
userStore.updateUser({
      pk: id,
      google_calendar_settings: {
        ...user.google_calendar_settings,
        oncall_schedules_to_consider_for_shift_swaps: value,
      },
    });
```